client_source/sqlplus/ut_run.bat

```diff
@@ -0,0 +1,16 @@
+@echo off
+
+set clientDir=%~dp0
+set projectDir=%__CD__%
+
+if not "%clientDir%" == "%clientDir: =%" (
+    echo Error: ut_run script path cannot have spaces.
+    exit /b 1
+)
+
+if "%1" == "" (
+    echo Usage: ut_run user/password@database [options...]
+    exit /b 1
+)
+
+sqlplus /nolog @"%clientDir%\ut_run.sql" '%clientDir%' '%projectDir%' %*
```

Comment:
since we now wrap the sqlplus call (user does not call sqlplus but call bash/cmd script instead), could we make the scripts in a way that they can run using both sqlplus and sqlcl?
could `sqlplus` / `sqlcl` be parameters?